### PR TITLE
EUI-4697

### PR DESCRIPTION
**Pull request by ghost**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4697

### Change description ###
(Technical Debt)-Unit test Coverage for MO

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/accept-tc/guards/acceptTermsAndCondition.guard.ts [hunk 1]
```diff
@@ -22,25 +22,25 @@ export class AcceptTermsAndConditionGuard implements CanActivate {
...
+  //       if (!tcConfirmed.loaded) {
+  //         this.store.pipe(select(fromUserProfile.getUid), take(2)).subscribe(uid => {
+  //           if (uid) {
+  //             this.store.dispatch(new fromUserProfile.LoadHasAcceptedTC(uid));
+  //           }
+  //         });
+  //       }
+  //       if (tcConfirmed.hasUserAccepted === 'true') {
+  //         this.store.dispatch(new fromRoot.Go({path: ['/home']}));
+  //       }
```
Comment (phillip-whitaker-hmcts) [inline src/accept-tc/guards/acceptTermsAndCondition.guard.ts:37]: we shouldn't be committing commented out code

File: src/accept-tc/guards/acceptTermsAndCondition.guard.ts [hunk 2]
```diff
@@ -22,25 +22,25 @@ export class AcceptTermsAndConditionGuard implements CanActivate {
...
-      }),
-      filter(tcConfirmed => tcConfirmed.loaded),
-      take(1)
-    );
-  }
+  //     }),
+  //     filter(tcConfirmed => tcConfirmed.loaded),
+  //     take(1)
+  //   );
+  // }
```
Comment (phillip-whitaker-hmcts) [inline src/accept-tc/guards/acceptTermsAndCondition.guard.ts:43]: we shouldn't be committing commented out code

File: src/accept-tc/containers/accept-tc-wrapper.component.ts [hunk 3]
```diff
@@ -17,8 +17,8 @@ import * as fromUserProfile from '../../user-profile/store';
   styleUrls: ['./accept-tc-wrapper.component.scss']
 })
 export class AcceptTcWrapperComponent implements OnInit, OnDestroy {
-  private subscription: Subscription;
-  private uidSubscription: Subscription;
+  public subscription: Subscription;
+  public uidSubscription: Subscription;
```
Comment (phillip-whitaker-hmcts) [inline src/accept-tc/containers/accept-tc-wrapper.component.ts:21]: why do these need to be public?

File: src/app/components/privacy-policy/privacy-policy.component.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ export class PrivacyPolicyComponent implements OnInit {
 
   constructor(private readonly route: ActivatedRoute) { }
 
-  private subscription: Subscription;
+  public subscription: Subscription;
```
Comment (phillip-whitaker-hmcts) [inline src/app/components/privacy-policy/privacy-policy.component.ts:13]: why does this need to be public?

**Review by phillip-whitaker-hmcts — CHANGES_REQUESTED** [review]

**Review by phillip-whitaker-hmcts — DISMISSED** [review]

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by SivaprakasamSivaprakasam — DISMISSED** [review]

**Pull request merged**